### ISSUE TITLE
notify_entries doesn't deliver all notifications to slack

### ISSUE THREAD

**Issue by girtsl** [Flexget/Flexget#1575]
### Expected behaviour:
A notification for every accepted entry is delivered to the chosen notifier(s), in my case - slack.

### Actual behaviour:
Some notifications are not delivered. These are the received slack notifications for 10 accepted entries (S02E01-S2E10):
```
incoming-webhook BOT [1:14 PM]  
The Man in the High Castle (2015) S02E01

[1:15]  
The Man in the High Castle (2015) S02E02


----- Today December 18th, 2016 -----

incoming-webhook BOT [10:14 AM]  
The Man in the High Castle (2015) S02E04


incoming-webhook BOT [1:14 PM]  
The Man in the High Castle (2015) S02E06


incoming-webhook BOT [10:14 PM]  
The Man in the High Castle (2015) S02E09

[10:15]  
The Man in the High Castle (2015) S02E10
```
The omissions seem random. Also, I can't seem to trigger a notification push when executing the task manually.

### Steps to reproduce:
Haven't figured out any steps that would reproduce this issue.

#### Config:
```
secrets: flexget-secrets.yml

templates:
  tv:
    notify_entries:
      to:
        - slack:
            web_hook_url: '{{ secrets.slack.webhook_url }}'

tasks:
  GET_SHOWS:
    template: tv
```
  
#### Log:
Even when the task above is executed with `--loglevel debug` and it accepts an entry, I can't find anything in the log about notifications, or errors.

### Additional information:

- Flexget Version: 2.8.6
- Python Version: 2.7.9
- Installation method: pip
- OS and version: OSMC

**Comment by liiight** [comment]
can you paste debug logs of task please?

**Comment by gazpachoking** [comment]
Bet this is the same as #1581 , and it's missing entries during tasks that rerun.

**Comment by liiight** [comment]
Good call...

**Comment by girtsl** [comment]
Sorry, for not getting back earlier. Indeed, this seems to be the same issue as #1581.
```
osmc@osmc:~/.flexget$ grep 'task_exit' flexget.log
2016-12-18 17:12 DEBUG    task          GET_SHOWS       not running task_exit yet because task will rerun
2016-12-18 17:12 DEBUG    task          GET_SHOWS       not running task_exit yet because task will rerun
```

**Comment by liiight** [comment]
Duplicate then, will be fixed in 2.9.0
